solvation_analysis/analysis.py

```diff
@@ -0,0 +1,103 @@
+# from abc import ABC
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+from MDAnalysis.analysis.rdf import InterRDF
+import numpy as np
+
+
+class Solute:
```

Comment:
This doesn't only encompass a solute, but rather a set of solute-solvent pairs? Perhaps we can work on the name down the track.

Reply:
It does encompass a set of solvent-solute pairs but I imagine that a user would almost never want 2 instances for the same solute. I imagine it will work best when the union of paired solvents is all atoms except the solute. With that in mind, I think Solute works nicely. However, it may not fit for some cases so I am happy to workshop it as things shape up.